### ISSUE TITLE
Can not find the witgen subcommand

### ISSUE THREAD

**Issue by developerworks** [bnjjj/witgen#27]
run `cargo install cargo-witgen` In MacOS:

```
➜  my_wit git:(main) ✗ cargo witgen
error: no such subcommand: `witgen`
```

I rename `~/.cargo/bin/cargo_witgen`  to `~/.cargo/bin/cargo-witgen`, and then ok.

**Comment by willemneal** [comment]
Thanks!  It was my bad.  I split the crate into a lib and a bin and named the bin incorrectly so cargo didn't pick it up.

Until the fix is published using it directly should work e.g. `

```
cargo_witgen
```

**Comment by bnjjj** [comment]
published under `v0.13` you can force install to update to the last `cargo-witgen`  using `cargo install -f cargo-witgen`
